src/main/resources/templates/bookstore.html

```diff
@@ -3,44 +3,84 @@
 <head>
     <meta charset="UTF-8">
     <title>Book Store</title>
+    <link th:rel="stylesheet" th:href="@{/webjars/bootstrap/4.0.0-2/css/bootstrap.min.css} "/>
 </head>
 <body>
-Sort By:
-
-<form th:action="@{/bookstore}">
-    <input type="radio" id="title" th:name="sort" value="title">
-    <label for="title">Title</label><br>
-    <input type="radio" id="author" th:name="sort" value="author">
-    <label for="author">Author</label><br>
-    <input type="radio" id="publisher" th:name="sort" value="publisher">
-    <label for="publisher">Publisher</label>
-    <input type="submit" value="Submit">
-</form>
-
-<a th:href="@{/logout}">Logout</a>
-<table>
-    <thead>
-    <tr>
-        <th> Title</th>
-        <th> ISBN</th>
-        <th> Author</th>
-        <th> Publisher</th>
-        <th> Inventory</th>
-    </tr>
-    </thead>
-    <tbody>
-    <tr th:if="${books.empty}">
-        <td colspan="2"> No Books Available</td>
-    </tr>
-    <tr th:each="book : ${books}">
-        <td><span th:text="${book.title}"> Title </span></td>
-        <td><span th:text="${book.isbn}"> ISBN </span></td>
-        <td><span th:text="${book.author}"> Author </span></td>
-        <td><span th:text="${book.publisher}"> Publisher </span></td>
-        <td><span th:text="${book.inventory}"> Inventory </span></td>
-        <td><span><a th:href="'/viewBook/'+${book.isbn}">View it</a></span></td>
-    </tr>
-    </tbody>
-</table>
+
+<div class="container-fluid">
+
+    <div class="row">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light" style="width:100%">
+            <a class="navbar-brand" href="/">Amazin</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+                    aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <ul class="navbar-nav mr-auto">
+                    <li class="nav-item" style="padding-right:50px;padding-left:50px">
+                        <a class="nav-link active" href="/bookstore">Visit Book Store</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="/uploadBook">Upload a book</a>
+                    </li>
+
+                </ul>
+
+                <form class="form-inline my-2 my-lg-0" th:action="@{/search}" method="get">
+                    <input class="form-control mr-sm-2" type="search" th:name="title" placeholder="Search"
+                           aria-label="Search">
+                    <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+                </form>
+
+            </div>
+        </nav>
+    </div>
+
```

Comment:
Can you get rid of the unnecessary whitespaces everywhere

Reply:
Sure thing, mb